config/settings/base.py

```diff
@@ -335,3 +338,19 @@
 SPADE_PROCESS_EXECUTORS: dict[str, object] = {}
 
 SPADE_HISTORY_PROVIDERS: dict[str, object] = {}
+
+SPADE_PERMISSIONS = rules.rulesets.RuleSet()
```

Comment:
@zak-nuccio this should be a string, referring to a name of a class. That class should be dynamically imported, and should have a method that is called upon import (or possibly `__init__`) where the actual permissions setup is happening. 
The actual configuration in config should look more like this:
```
SPADE_PERMISSION_MANAGER = env("SPADE_PERMISSION_MANAGER", default="spadeapp.utils.permissions.DefaultPermissionManager")
```